REL/training_datasets.py

```diff
@@ -37,7 +37,7 @@ def load(self):
 
             print("Loading {}".format(ds))
             datasets[ds] = self.__read_pickle_file(
-                "{}/generated/test_train_data/{}.pkl".format(self.base_url, ds)
+                os.path.join(self.base_url, 'generated/test_train_data/', ds) + '.pkl'

```

Comment:
Inconsistency; `f"{ds}.pkl"` is probably neater

Reply:
Agreed!